src/mujincontrollerclient.cpp

```diff
@@ -332,7 +332,7 @@ void ObjectResource::LinkResource::GetGeometries(std::vector<ObjectResource::Geo
                 ObjectResource::GeometryResourcePtr geometry(new GeometryResource(controller, this->objectpk, GetJsonValueByKey<std::string>(*it, "pk")));
                 geometry->linkpk = linkpk;
                 LoadJsonValueByKey(*it,"name",geometry->name,geometry->pk);
-                LoadJsonValueByKey(*it,"visible",geometry->visible);
+                LoadJsonValueByKey(*it,"visible",geometry->visible,true);
```

Comment:
Should we also do this above? [This reference](https://github.com/mujin/controllerclientcpp/blob/44ed7220bda86d1faccd1d804780adabf848e90c/src/mujincontrollerclient.cpp#L296).

Reply:
yes.

another possibility is setup https://github.com/mujin/controllerclientcpp/blob/master/include/mujincontrollerclient/mujincontrollerclient.h#L789 to `bool visible = true;` statically, but I fear if it breaks Windows build. That's why I tried to introduce https://github.com/mujin/controllerclientcpp/pull/146

Reply:
tested https://github.com/mujin/controllerclientcpp/actions/runs/8889962316

it is safe to modify header, so I changed the implementation.